internal/host/hostcommands/instruction_manager.go

```diff
@@ -51,9 +51,9 @@ type InstructionConfig struct {
 	ServiceBaseURL           string            `envconfig:"SERVICE_BASE_URL"`
 	ServiceCACertPath        string            `envconfig:"SERVICE_CA_CERT_PATH" default:""`
 	ServiceIPs               string            `envconfig:"SERVICE_IPS" default:""`
-	InstallerImage           string            `envconfig:"INSTALLER_IMAGE" default:"quay.io/ocpmetal/assisted-installer:latest"`
-	ControllerImage          string            `envconfig:"CONTROLLER_IMAGE" default:"quay.io/ocpmetal/assisted-installer-controller:latest"`
-	AgentImage               string            `envconfig:"AGENT_DOCKER_IMAGE" default:"quay.io/ocpmetal/assisted-installer-agent:latest"`
+	InstallerImage           string            `envconfig:"INSTALLER_IMAGE" default:"quay.io/example/assisted-installer:latest"`
```

Comment:
why example?

Reply:
Because I made a mistake :sweat_smile: 

I was using example in the tests just because it should never matter so I figured it was better to make it explicit, but this isn't a test so I guess I used the wrong find/replace command.